_episodes/03-benchmark-and-scaling.md

```diff
@@ -1,17 +1,20 @@
 ---
 title: "Benchmarking and Scaling"
-teaching: 0
-exercises: 0
+teaching: 15
+exercises: 15
 questions:
 - "What is benchmarking?"
-- "How do I benchmark an application?"
+- "How do I do a benchmark in LAMMPS?"
```

Comment:
If we can leave this general I would prefer it, as then it can be used for other applications with minimal editing